src/tests/build.sh

```diff
@@ -432,6 +439,11 @@ usage_list+=("-buildtestwrappersonly: only build the test wrappers.")
 usage_list+=("-copynativeonly: Only copy the native test binaries to the managed output. Do not build the native or managed tests.")
 usage_list+=("-generatelayoutonly: only pull down dependencies and build coreroot.")
 
+usage_list+=("-test:xxx - only build a single test project");
+usage_list+=("-dir:xxx - build all tests in a given directory");
+usage_list+=("-tree:xxx - build all tests in a given subtree");
+usage_list+=("-all:xxx - rebuild all the specified tests");
```

Comment:
`-all` has no argument:
```suggestion
usage_list+=("-all - rebuild all the specified tests");
```